config/README.md

```diff
@@ -1,8 +1,15 @@
 # Kustomizations for Baremetal Operator
 
 This folder contains kustomizations for the Baremetal Operator. They have
-traditionally been used through the [deploy.sh](../tools/deploy.sh) script,
+been used through the [deploy-cli](../tools/deploy-cli) library,
```

Comment:
nit: "They can be used". This is new stuff, not something we have been doing :smile: 
I would also leave the note about deploy.sh, because that is indeed how they have been used traditionally (and still are in metal3-dev-env).